modules/runtime/nuxeo-runtime-launcher/src/test/java/org/nuxeo/log4j2/TestMaskSensitiveDataRewritePolicy.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * (C) Copyright 2021 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Kevin Leturc <kleturc@nuxeo.com>
+ */
+
+package org.nuxeo.log4j2;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+/**
+ * @since 11.5
+ */
+public class TestMaskSensitiveDataRewritePolicy {
+
+    protected MaskSensitiveDataRewritePolicy policy = new MaskSensitiveDataRewritePolicy();
+
+    @Test
+    public void testMaskSensitiveAWS() {
+        String testStr = "Should replace this AKIAI53OIMNYFFMFQX5K key";
```

Comment:
Shouldn't we choose keys that look more fake like AKIATEST1234ABC to avoid alerts on security bots that parse sources?

Reply:
I think we really need a real key, especially for AWS keys as the Pattern is made to catch only AWS keys.

Do you know such bots? We can probably add something like comments/suppress warnings to avoid that.

Reply:
We can have a test value that's real according to the pattern but still contains something like TEST I think.